package Exception;

public class ThrowException {
    public static void main(String[] args) {
        throw new ArithmeticException("divide by zero");
    }
}
